controls/radofficenavigationbar/view-modes.md

```diff
@@ -0,0 +1,43 @@
+---
+title: View Modes
```

Comment:
I would use Display Modes instead of View Modes, if possible. This is because at first look I've read View Mode**l**s, which was a bit confusing.